src/types.rs

```diff
@@ -31,15 +31,27 @@ pub struct BalanceResponse {
     #[doc = " Total computed balance"]
     pub value: u64,
 }
+#[doc = " The value of the Crossover and the blinder"]
+#[derive(Clone, PartialEq, Debug, Deserialize, Serialize)]
+pub struct Crossover {
```

Comment:
This needs to be changed imo (as well for the crossover struct in tx.rs)
We should not have a crossover struct from Phoenix and then use that struct as a field in another struct (with additional fields) with the name crossover as well.

Reply:
I agree about that because its also super confusing. I had to set `use wallet_core::tx::Crossover as WasmCrossover`